### ISSUE TITLE
How is this suppose to work?

### ISSUE THREAD

**Issue by TomR** [ejyothi/MantisBT-SuggestUsername#1]
The autocomplete does not seem to work.

Error in Chrome console:

```
Cannot read property 'autocomplete' of undefined
    at HTMLInputElement.<anonymous> 
```

**Comment by libregeek** [comment]
Can you please let me know which version of Mantis and the Plugin are you using?

**Comment by TomR** [comment]
I am using MantisBT 2.2.0 and Ajax User Monitoring 0.2.

However I guess my problem is not hist with your plugin, but is with my installation. Files from the plugin 'files' folder seems not to be found! This is a problem with all my plugins. ( So images are not shown and .css and .js are probably not loaded! ).

plugin_fil.php is not able to find files in <Plugin>/files folder??
If I show URL directly http://my.domain.com/plugin_file.php?file=<Plugin folder>/<file from 'plugin/files' folder>, it is NOT found.

Do you have any idea?

**Comment by TomR** [comment]
It is a bit more complicated as I found out during testing.

Files in the 'plugin/files' folder can be found, how ever images are not shown.
If they are .ccs files I can see them in the browser. The same with .js files. But images are NOT shown. However it seems the .css and .js are not loaded?? And that wil case the error?

How can I check if .js and .css files of the plugin are loaded?

**Comment by libregeek** [comment]
This plugin doesn't work with MantisBT-2.2. This is because jQueryUI is removed in 2.2 and this plugin requires jQueryUI. This works in 1.3, 2.0 and 2.1. I guess the mantis community is fixing their old jQueryUI plugin to work with 2.3. So you have two choices - either go back to Mantis-2.1 or wait for Mantis-2.3. A third option is to fork this plugin and manually add jQueryUI files.

**Comment by TomR** [comment]
Do not make enay effort in solving this. Your plugin is working correct, On a different website is working as intended. So the problem is with with my seup of this particular MantisBT instance.

How ever any ideas are well appreciated.

Sorry to have bothered you. ;-)

**Comment by TomR** [comment]
I now noticed your comment about JQueryUI not available in MantisBT V2.2 ;-)
You are correct, I tested on different server with only 2.0.0 installed.

Thanks for your explanation!
